From 11da9f0c6d145e482991d29a771ce717d2f1b92b Mon Sep 17 00:00:00 2001
Message-Id: <11da9f0c6d145e482991d29a771ce717d2f1b92b.1649663748.git.geliang.tang@suse.com>
From: Yang Li <yang.lee@linux.alibaba.com>
Date: Mon, 8 Feb 2021 18:30:13 +0800
Subject: [PATCH] selftests/bpf: Remove unneeded semicolon
Git-commit: 11da9f0c6d145e482991d29a771ce717d2f1b92b
Patch-mainline: v5.12-rc2
References: bsn#127
Modified-by-SEL: No


Eliminate the following coccicheck warning:
./tools/testing/selftests/bpf/test_flow_dissector.c:506:2-3: Unneeded
semicolon

Reported-by: Abaci Robot <abaci@linux.alibaba.com>
Signed-off-by: Yang Li <yang.lee@linux.alibaba.com>
Signed-off-by: Andrii Nakryiko <andrii@kernel.org>
Link: https://lore.kernel.org/bpf/1612780213-84583-1-git-send-email-yang.lee@linux.alibaba.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/testing/selftests/bpf/test_flow_dissector.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/bpf/test_flow_dissector.c b/tools/testing/selftests/bpf/test_flow_dissector.c
index 01f0c634d548..571cc076dd7d 100644
--- a/tools/testing/selftests/bpf/test_flow_dissector.c
+++ b/tools/testing/selftests/bpf/test_flow_dissector.c
@@ -503,7 +503,7 @@ static int do_rx(int fd)
 		if (rbuf != cfg_payload_char)
 			error(1, 0, "recv: payload mismatch");
 		num++;
-	};
+	}
 
 	return num;
 }
-- 
2.34.1

